src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBMetricsProvider.cs

```diff
@@ -17,6 +17,9 @@ internal class CosmosDBMetricsProvider
         private readonly Container _monitoredContainer;
         private readonly Container _leaseContainer;
         private readonly string _processorName;
+        private int _assignWorkerOnNotFoundCount = 0;
+        private int _maxAssignWorkerOnNotFoundCount = 5;
```

Comment:
```suggestion
        private readonly int _maxAssignWorkerOnNotFoundCount = 5;
```

And move it with the other readonly variables?